programmers/2일차/가운데-글자-가져오기.js

```diff
@@ -0,0 +1,16 @@
+function solution(s) {
+    let answer = "";
+    let splitS = s.split("");
```

Comment:
```suggestion
     const words = s.split("");
     let answer = "";
```
기존의 splitS 변수는  밑에 코드에서 변수 값 자체가 바뀌지 않기 때문에  상수로 선언을 해줘야 합니다 
따라서 const를 써주시는게 좋아요  변수명은  splitS 보다는 words가  좀 더 직관적인 것 같아서 고쳤습니다 ㅋㅋ
저도  변수명 짓는게 정말 어려워요 이게 최선이 아닐수도 있지만 ....